cli/src/api.ts

```diff
@@ -90,6 +91,7 @@ export type PlaneTerminationReason =
   | 'key_expired'
   | 'lost'
   | 'startup_timeout'
+  | 'internal-error'
```

Comment:
maybe we should resolve the underscore/hyphen discrepancy before we go too far down this road? I'm fine with either as long as we are consistent.

Reply:
Whoops - these should all be underscored - fixed!